rslib/src/storage/deck/mod.rs

```diff
@@ -23,7 +24,7 @@ fn row_to_deck(row: &Row) -> Result<Deck> {
     let id = row.get(0)?;
     Ok(Deck {
         id,
-        name: row.get(1)?,
+        name: NativeDeckName(row.get(1)?),
```

Comment:
Similarly, are names from the database guaranteed to be normalised?

Reply:
Even if Anki never had bugs, we can't prevent external apps or add-ons from modifying the DB